docs/api/covidcast-signals/chng.md

```diff
@@ -26,6 +26,8 @@ commercial purposes.
 | --- | --- |
 | `smoothed_outpatient_covid` | Estimated percentage of outpatient doctor visits with confirmed COVID-19, based on Change Healthcare claims data that has been de-identified in accordance with HIPAA privacy regulations, smoothed in time using a Gaussian linear smoother |
 | `smoothed_adj_outpatient_covid` | Same, but with systematic day-of-week effects removed; see [details below](#day-of-week-adjustment) |
+| `smoothed_outpatient_cli` | Estimated percentage of outpatient doctor visits primarily about COVID-related symptoms, based on Change Healthcare claims data that has been de-identified in accordance with HIPAA privacy regulations, smoothed in time using a Gaussian linear smoother |
```

Comment:
```suggestion
| `smoothed_outpatient_cli` | Estimated percentage of outpatient doctor visits primarily about COVID-related symptoms, based on Change Healthcare claims data that has been aggregated in accordance with HIPAA privacy regulations, smoothed in time using a Gaussian linear smoother |
```

I feel this would be better, but if CH has specific language choices then ignore it.

Reply:
alas it's their language 😞

Reply:
No worries, :) looks good to go then